curriculum/challenges/italian/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fac56271087806def55b33.md

```diff
@@ -26,7 +26,7 @@ assert.equal(document.querySelector('fieldset:nth-child(3) > label:nth-child(3)'
 Dovresti posizionare il testo prima dell'elemento `select`.
 
 ```js
-assert.match(document.querySelector('fieldset:nth-child(3) > label:nth-child(3)')?.innerHTML?.replace(/[\t\n]+/g, ''), /^How did you hear about us\?/);
+assert.match(document.querySelector('fieldset:nth-child(3) > label:nth-child(3)')?.innerHTML?.trim().replace(/[\t\n]+/g, ''), /^How did you hear about us\?/);
```

Comment:
You don't have to change the test for other languages, you should only change English.

Reply:
I have restored this file to its original state.